src/Domain/ValueObjects/ShoppingItemFilterEnum.php

```diff
@@ -2,29 +2,46 @@
 
 namespace Lindyhopchris\ShoppingList\Domain\ValueObjects;
 
+use http\Exception\InvalidArgumentException;
+
 class ShoppingItemFilterEnum
 {
+
+    const ONE = 'all';
+    const TWO = 'only completed';
```

Comment:
Suggestion: good job on adding these constants, and you have referred to them correctly in the class i.e. `self::ONE` is the correct way to reference them in code.

There's a few changes we'd normally request here:

1. Our coding standards say all constants MUST have a visibility, i.e. `public`, `protected` or `private`. In this case they would be `public`.
2. The constant should accurately describe what it is. So instead of `ONE` it should be `ALL`. Instead of `TWO` it should be `ONLY_COMPLETED`. Etc.
3. Although the contents of the string can be anything, typically a dev would use snake case - e.g. `only_completed` - or dash-case - e.g. `only-completed` - for the string value. It works with the spaces, it's just considered _nicer_ if dash or snake case are used.